content/docs/integrations/kubernetes/index.md

```diff
@@ -97,6 +97,9 @@ You should see a page like this
       },
       "Action": "sts:AssumeRole",
       "Condition": {
+        "StringEquals": {
+          "sts:ExternalId": "XXXXYour ID XXXX"
```

Comment:
Does AWS generate the External ID for the user or where does it come from?

We use text like `ACCOUNT ARN SUPPLIED ON THE CONFIGURATION PAGE` above and below this line to indicate the place to replace the ARN and the role from AWS. Would it make sense to use the same style of text here instead of `XXXXYour ID XXXX`.

Reply:
The External ID is user defined.

Reply:
Done, take a look now, let me know what you think.